### DISCUSSION TITLE
am unable to push my code over the repo reasion behind  ! [remote rejected] master -> master (push declined due to email privacy restrictions)

### DISCUSSION THREAD

**Discussion by Suhem-Ahmed** [community/community#132275] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

i add all the emails and clear all security checks 
can anyone help me out!

**Comment by davevad93** [comment, marked as answer]
Hi @Suhem-Ahmed ,

Go to `Settings` => `Emails` and uncheck the following option:

![image](https://github.com/user-attachments/assets/d3bc2fae-9930-411a-a24a-cf05926242a1)

If by doing that you're still unable to push from the command line, you should reconfigure your global `user.email` using the `@users.noreply.github.com` which can be found in the `Emails` settings.

**Reply by Suhem-Ahmed** [reply]
Thanks @davevad93 
Solved 👍

**Reply by davevad93** [reply]
You're welcome @Suhem-Ahmed !
